packages/graphiql-toolkit/docs/create-fetcher.md

```diff
@@ -184,7 +184,8 @@ const fetcher = createGraphiQLFetcher({
 
 export const App = () => <GraphiQL fetcher={fetcher} />;
 
-ReactDOM.render(document.getElementByID('graphiql'), <App />);
+const root = createRoot(document.getElementById('graphiql'));
```

Comment:
I realized only now that previous code snippet was incorrect, the component should be in 1st argument

we had a lot of mistakes 😅
<img width="1007" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/f10acf8c-7504-4666-90b9-b40995725eab">